### PR TITLE
Doc: Update PT tensor fitting

### PR DESCRIPTION
**Pull request by anyangml**

### CODE REVIEW

File: doc/backend.md [hunk 1]
```diff
@@ -23,18 +23,18 @@ DeePMD-kit does not use the TensorFlow v2 API but uses the TensorFlow v1 API (`t
 [PyTorch](https://pytorch.org/) 2.0 or above is required.
 While `.pth` and `.pt` are the same in the PyTorch package, they have different meanings in the DeePMD-kit to distinguish the model and the checkpoint.
 
-### DPModel {{ dpmodel_icon }}
+### DP {{ dpmodel_icon }}
```
Comment (njzjz) [inline doc/backend.md:26]: Other places also need to update, for example https://deepmodeling--3385.org.readthedocs.build/projects/deepmd/en/3385/model/train-se-e2-a.html
Please do a global search in the doc directory.

File: doc/model/train-fitting-tensor.md [hunk 2]
```diff
@@ -1,16 +1,33 @@
-# Fit `tensor` like `Dipole` and `Polarizability` {{ tensorflow_icon }}
+# Fit `tensor` like `Dipole` and `Polarizability` {{ tensorflow_icon }} {{ pytorch_icon }} {{ dpmodel_icon }}
 
 :::{note}
-**Supported backends**: TensorFlow {{ tensorflow_icon }}
+**Supported backends**: TensorFlow {{ tensorflow_icon }} {{ pytorch_icon }} {{ dpmodel_icon }}
```
Comment (njzjz) [inline doc/model/train-fitting-tensor.md:4]:
```suggestion
**Supported backends**: TensorFlow {{ tensorflow_icon }}, PyTorch {{ pytorch_icon }}, DP {{ dpmodel_icon }}
```

File: doc/model/train-fitting-tensor.md [hunk 3]
```diff
@@ -81,9 +102,48 @@ The JSON of `polar` type should be provided like
...
+
+:::{tab-item} PyTorch {{ pytorch_icon }}
+
+The JSON of `dipole` type should be provided like
+```json
+	"atom_exclude_types": [
+      1
+    ],
+	"fitting_net" : {
+		"type": "dipole",
```
Comment (njzjz) [inline doc/model/train-fitting-tensor.md:115]: https://deepmodeling--3385.org.readthedocs.build/projects/deepmd/en/3385/train/train-input.html#model-standard-fitting-net-dipole

We may remove `
(Supported Backend: TensorFlow)`

Comment (njzjz) [reply doc/model/train-fitting-tensor.md:115]: Also for polar

File: doc/model/train-fitting-tensor.md (hunk 2 above)
Comment (njzjz) [reply doc/model/train-fitting-tensor.md:4]: this seems not fixed?

**Review by njzjz — COMMENTED** [review]
Need to update: https://deepmodeling--3385.org.readthedocs.build/projects/deepmd/en/3385/model/dprc.html
https://deepmodeling--3385.org.readthedocs.build/projects/deepmd/en/3385/model/train-energy.html

**Comment by anyangml** [comment]
> Need to update: https://deepmodeling--3385.org.readthedocs.build/projects/deepmd/en/3385/model/dprc.html https://deepmodeling--3385.org.readthedocs.build/projects/deepmd/en/3385/model/train-energy.html

I am quit sure what caused this.

**Comment by anyangml** [comment]
> Need to update: https://deepmodeling--3385.org.readthedocs.build/projects/deepmd/en/3385/model/dprc.html https://deepmodeling--3385.org.readthedocs.build/projects/deepmd/en/3385/model/train-energy.html

what need to be updated here?

**Review by njzjz — APPROVED** [review]

**Review by iProzd — APPROVED** [review]

**Pull request merged**